tests/api.c

```diff
@@ -13150,6 +13151,16 @@ static int test_wc_AesCbcEncryptDecrypt (void)
         } else {
             cbcE = WOLFSSL_FATAL_ERROR;
         }
+#ifdef WOLFSSL_AES_CBC_ALIGNMENT_CHECKS
+        if (cbcE == 0) {
+            cbcE = wc_AesCbcEncrypt(&aes, enc, vector, sizeof(vector) - 1);
```

Comment:
I guess we cannot add this in all cases for the `wc_AesCbcEncrypt` because the logic was missing. Maybe Todd or Sean knows some history there.

Reply:
We _could_ add a test when `!defined(WOLFSSL_AES_CBC_ALIGNMENT_CHECKS)` to expect success on unaligned length.